src/data/firebase.ts

```diff
@@ -0,0 +1,106 @@
+import { getImportedKey } from "./index";
+import * as firebase from "firebase/app";
+import "firebase/firestore";
```

Comment:
How big is this?

Reply:
~320kb minified, ~90kb gzipped

Reply:
this is now lazy loaded and i verified it in dev. will double check it when ci builds